utils/forest_utils/src/net/http.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright 2019-2022 ChainSafe Systems
```

Comment:
It'd be nice to add some method/trait documentation. It's hard to understand what are those constructs for without going deeper into the code.

Reply:
Done